index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
```

Comment:
Please refer to the data model(s) by there media types... this addresses the ambiguity, and provides a clean entry point into the security considerations and normative requirements associated with this sentence... the term "conforming document" should be equally acceptable, but it hides the JSON-LD requirement that the media types make explicit.

Reply:
Transforming data model vs transforming the format?

We can have the same data model in multiple formats (JSON/JSON-LD, XML, ...). In this case the claim names and definitions/vocabularies are preserved. IMO this should not be an issue since all the information is preserved.

If there's a data model transformation, you can have 2 data models in a same or different format. In this case a mapping between the claims and definitions is required. This may be bit tricky if different definitions/vocabularies are used.

Reply:
DIDs had an abstract data model... currently VCs only support JSON-LD.

I'm not sure I understand your question... Obviously if you start with JSON-LD, you might be able to round trip it through other content types... but you won't ever express things that can't be expressed in JSON, in JSON-LD.

Examples of this from DID Core... you can't preserve `@context` when converting to XML because of XML rules... so you need to define a transformation to XML... and you can't preserve integer tags from CBOR in JSON-LD.

Given how badly DID Core ended wrt the abstract data model, I don't think we should repeat "undocumented transformation rules allow for the data model to be whatever you want" arguments.

I would object to attempting to define transformation rules in this working group, its harmful complexity, and its not necessary to achieve interop on the media types we request registration for that express the core data model.

It seems reasonable for W3C to try to do a good job at data models that are `+ld+json`... and leave data models that are not, to working groups that have experts in other media types.

Reply:
@OR13 I have made it explicit which media types are allowed for a `conforming document` in bd9aab2a283bd45789984fe2fc528c696ebe2aac.

Reply:
> [@OR13] Please refer to the data model(s) by [their] media types

Data models don't have media types.

Data models may be _expressed_ in a variety of media types.

Sometimes expression in one or more specific data types is required for interop, but the "anyone can say anything about anything" mantra gets in the way of forbidding expression in other media types, not to mention that lossless round-trips make storage in other media types also acceptable.